scss/ionic.app.scss

```diff
@@ -21,3 +21,6 @@ $ionicons-font-path: "../lib/ionic/fonts" !default;
 // Include all of Ionic
 @import "www/lib/ionic/scss/ionic";
 
+// Application Styles
+@import "www/scss/rcredits";
```

Comment:
I think the idea is that you put all of your scss files right in the scss dir, not under www. That may be why live update isn't working. Please see the mealyzer-mobile app as an example. https://github.com/hooverlunch/mealyzer-mobile